spec/unit/appmgmt_spec.rb

```diff
@@ -46,7 +46,7 @@ def compile_to_env_catalog(string)
 
       application app {
         prod { one: host => ahost, export => Cap[cap] }
-        cons { two: host => ahost, consume => Cap[cap] }
```

Comment:
You are testing two things with one test; a) that consume works and b) that missing parameters works - seems best to have two separate test cases.